module/callback_handlers.py

```diff
@@ -80,6 +80,7 @@ def informative_callback(update: Update, context: CallbackContext) -> None:
         index_key: int = list(ON_DEMAND_TEXTS.values()).index(cmd)
         message_text: str = get_on_demand_text(locale, list(ON_DEMAND_TEXTS.keys())[index_key])
     else:
+        cmd = cmd if cmd != "report" else "segnalazione"
```

Comment:
you save a useless re-assignment in this way
```suggestion
         if cmd == "report":
             cmd = "segnalazione" 
```